apps/web/client/src/app/project/[id]/_components/right-panel/chat-tab/chat-messages/message-content/tool-call-simple.tsx

```
@@ -192,6 +195,18 @@ export function ToolCallSimple({
                     return 'Reading Onlook instructions';
                 case TYPECHECK_TOOL_NAME:
                     return 'Checking types';
+                case CLONE_WEBSITE_TOOL_NAME:
+                    const params13 = toolInvocation.input as z.infer<typeof CLONE_WEBSITE_TOOL_PARAMETERS>;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)